src/components/tabController/tabController.api.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "TabController",
```

Comment:
TabController.api.json file exist under apis directory. Please remove this duplication from PR